### PR TITLE
[1262] Add link to edit training location

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
Edit training locations

### Changes proposed in this pull request
Add link to edit training locations

### Guidance to review
# After
localhost_3000_organisations_2AT_locations

!DO NOT MERGE!

### CODE REVIEW

**Comment by benilovj** [comment]
If we make the link conditional for opted-in providers, then it can be mergeable.

**Comment by defong** [comment]
[Link to Location Name](https://trello.com/c/hOP6TeGA/1262-link-to-location-name)

File: app/views/sites/index.html.erb [hunk 1]
```diff
@@ -3,6 +3,11 @@
 <% content_for :before_content do %>
   <nav class="govuk-breadcrumbs">
     <ol class="govuk-breadcrumbs__list">
+      <% if @has_multiple_providers then %>
```
[the remark this answers is not in the document]
Comment (fofr) [reply app/views/sites/index.html.erb]: Looks like the courses page on Rails has the wrong breadcrumb too

[the remark this answers is not in the document]
Comment (benilovj) [reply app/views/sites/index.html.erb]: I'll pop a card on the backlog for the courses page breadcrumb.

**Pull request merged**